full-stack-tests/core/src/frontend/standalone/ExtensionAdmin.test.ts

```diff
@@ -3,32 +3,32 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
-import { assert } from "chai";
-import { ExternalServerExtensionLoader, IModelApp } from "@bentley/imodeljs-frontend";
-import { ProcessDetector } from "@bentley/bentleyjs-core";
+// import { assert } from "chai";

```

Comment:
Planning to resurrect commented-out code in future?

Reply:
Removal was just merged into `master`. I'll merge now and clean this up.

Reply:
Merged and it has been removed.